src/SoftKiller.jl

```diff
@@ -0,0 +1,169 @@
+using JetReconstruction
+
+mutable struct SoftKiller <: TilingBase
+    _ymax::Float64
+    _ymin::Float64
+    _requested_drap::Float64
+    _requested_dphi::Float64
+    #selector 
+    _ntotal::Int64
+    _ngood::Int64
+    _dy::Float64
+    _dphi::Float64
+    _cell_area::Float64
+    _inverse_dy::Float64
+    _inverse_dphi::Float64
+    _ny::Int64
+    _nphi::Int64
+
+    function SoftKiller(rapmin::Float64, rapmax::Float64, drap::Float64, dphi::Float64)
+        print("4 variables \n")
+        grid = new(rapmax, rapmin, drap, dphi, 0, 0, 0.0, 0.0, 0.0, 0.0, 0.0, 0, 0)
+        _setup_grid(grid)
+        print(description(grid))
+        grid
+    end
+
+    function SoftKiller(rapmax::Float64, grid_size::Float64)
+        print("2 variables\n")
+        grid = new(rapmax, -rapmax, grid_size, grid_size, 0, 0, 0.0, 0.0, 0.0, 0.0, 0.0, 0,
+                   0)
+        _setup_grid(grid)
+        print(description(grid))
+        grid
+    end
+end
+
+tile_index(sk::SoftKiller,
+           p::PseudoJet)::Int64 = begin
```

Comment:
why does this have to be Int64 not just `Int`

Reply:
why does the type need to be specified at all?